.gitmodules

```diff
@@ -1,16 +1,15 @@
-[submodule "atmos_cubed_sphere"]
-  path = atmos_cubed_sphere
-  url = https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere
-  branch = dev/emc
 [submodule "ccpp/framework"]
   path = ccpp/framework
   url = https://github.com/NCAR/ccpp-framework
   branch = main
 [submodule "ccpp/physics"]
   path = ccpp/physics
-  url = https://github.com/NCAR/ccpp-physics
+  url = https://github.com/lisa-bengtsson/ccpp-physics
   branch = main
```

Comment:
You need to change the name of the branch to your ccpp-physics PR branch, prog_closure, otherwise this is telling git to grab the main branch on your fork.